core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/WorkspaceStatus.java

```diff
@@ -28,8 +28,9 @@
      * <p>Workspace becomes starting only if it was {@link #STOPPED}.
      * The status map:
      * <pre>
-     *  STOPPED -> <b>STARTING</b> -> RUNNING (normal behaviour)
-     *  STOPPED -> <b>STARTING</b> -> STOPPED (failed to start)
+     *  STOPPED -> <b>STARTING</b> -> RUNNING  (normal behaviour)
```

Comment:
This is a definition for the _STARTING_ status, there is now way you can go from STARTING to SNAPSHOTTING workspace, and only possible options are described in status doc, which are:
```
STOPPED -> STARTING -> RUNNING (regular flow)
STOPPED -> STARTING -> STOPPING (start cancellation)
STOPPED -> STARTING -> STOPPED (failed to start)
```